train.py

```diff
@@ -1,45 +1,13 @@
-"""
-This training script can be run both on a single gpu in debug mode,
-and also in a larger training run with distributed data parallel (ddp).
-
-To run on a single GPU, example:
-$ python train.py --batch_size=32 --compile=False
-
-To run with DDP on 4 gpus on 1 node, example:
-$ torchrun --standalone --nproc_per_node=4 train.py
-
-To run with DDP on 4 gpus across 2 nodes, example:
-- Run on the first (master) node with example IP 123.456.123.456:
-$ torchrun --nproc_per_node=8 --nnodes=2 --node_rank=0 --master_addr=123.456.123.456 --master_port=1234 train.py
-- Run on the worker node:
-$ torchrun --nproc_per_node=8 --nnodes=2 --node_rank=1 --master_addr=123.456.123.456 --master_port=1234 train.py
-(If your cluster does not have Infiniband interconnect prepend NCCL_IB_DISABLE=1)
-"""
-
-import math
-import os
-import pickle
-import time
-from contextlib import nullcontext
+from trainers.trainer import Trainer
 
-import numpy as np
-import torch
-import yaml
-from torch.nn.parallel import DistributedDataParallel as DDP
+USE_GPU = False
```

Comment:
Can we have this be configurable without editing the source? Something simple like the following should do the trick

```python
import os
from distutils.util import strtobool

USE_GPU = strtobool(os.environ.get("USE_GPU", "true"))
```

Then if we need to run without GPU we can just do `USE_GPU=false python train.py`

Reply:
done